### ISSUE TITLE
MSSQL

### ISSUE THREAD

**Issue by Stathis90** [reev-create/Directus-SQL-Panel#1]
I get the below error when running a query
![image](https://user-images.githubusercontent.com/15157178/176669732-ee3452a3-15eb-48d6-a38c-2b3b6d62eb20.png)

![image](https://user-images.githubusercontent.com/15157178/176669687-230ed429-07a9-4271-9e7d-56b41f26c6a7.png)

**Comment by reev-create** [comment]
Can you show the screenshot of the table structure dbo.tender ?

**Comment by reev-create** [comment]
Can you try 
`select * from directus_users`

**Comment by reev-create** [comment]
Wait let me re-check the code. It seems there's a mismatch of uuid of the panel while being created.

**Comment by Stathis90** [comment]
FYI I am using the latest firectus verion

**Comment by reev-create** [comment]
Yeah no worries, I'm using the latest version myself as well.

**Comment by reev-create** [comment]
Yes, that's what. I must have uploaded the older file. I'll compile the new one and commit in some time.

**Comment by Stathis90** [comment]
Hey! Have you found some time to recompile?

**Comment by reev-create** [comment]
Sorry about that! Got a lil busy this week. I'll compile that by E.O.W.

**Comment by Stathis90** [comment]
Thank you I'll wait

**Comment by Stathis90** [comment]
Any updates?
